packages/tocco-ui/src/Upload/Upload.stories.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */
```

Comment:
why is this need? usually  there is something wrong if you have to disable lint. or at least only disable a single rule